### PR TITLE
test(jwt-sample): adds tests to the jwt to show e2e flow works

### PR DESCRIPTION
**Pull request by jmcdo29**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Other... Please describe:
```

Adds tests for the JWT example, number 19

## What is the current behavior?

No tests
Issue Number: #5472 

## What is the new behavior?

There's tests!

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

## Other information

### CODE REVIEW

File: sample/19-auth-jwt/e2e/app/app.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  });
+
+  afterAll(async() => {
+    await app.close();
+  });
+
+  it('should get a JWT then successfully make a call', async () => {
+    const loginReq = await request(app.getHttpServer())
+      .post('/auth/login')
+      .send({username: 'john', password: 'changeme'})
```
Comment (kamilmysliwiec) [inline sample/19-auth-jwt/e2e/app/app.e2e-spec.ts:25]:
```suggestion
      .send({ username: 'john', password: 'changeme' })
```

File: sample/19-auth-jwt/e2e/app/app.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      imports: [AppModule],
+    }).compile();
+
+    app = modRef.createNestApplication();
+    await app.init();
+  });
+
+  afterAll(async() => {
+    await app.close();
+  });
```
Comment (kamilmysliwiec) [inline sample/19-auth-jwt/e2e/app/app.e2e-spec.ts:20]: Can we move this block to the end of the `before()` function?

Comment (jmcdo29) [reply sample/19-auth-jwt/e2e/app/app.e2e-spec.ts:20]: I assume you mean the end of the `describe` block?

Comment (kamilmysliwiec) [reply sample/19-auth-jwt/e2e/app/app.e2e-spec.ts]: Ugh I meant `describe` of course! Apologies for confusion

Comment (jmcdo29) [reply sample/19-auth-jwt/e2e/app/app.e2e-spec.ts]: No worries. Got it moved

File: sample/19-auth-jwt/e2e/app/app.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    app = modRef.createNestApplication();
+    await app.init();
+  });
+
+  it('should get a JWT then successfully make a call', async () => {
+    const loginReq = await request(app.getHttpServer())
+      .post('/auth/login')
+      .send({ username: 'john', password: 'changeme' })
+      .expect(201);
```
Comment (kamilmysliwiec) [inline sample/19-auth-jwt/e2e/app/app.e2e-spec.ts:22]:
```suggestion
      .expect(201);
      
```

**Pull request merged**